robot_smach_states/examples/test-detect-faces

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
```

Comment:
I see this is a test of the DetectFace state, but it is in the 'examples' directory. Is it an example, or a test?